.gitignore

```diff
@@ -8,4 +8,7 @@ workspace.code-workspace
 node_modules/
 
 #Intellij project file
-.idea
\ No newline at end of file
+.idea
```

Comment:
Nit: https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
I've got a global gitignore file, but tool-versions lets me pin the versions of things like Go in each project. Adding it to a global gitignore file prevents me from seeing changes to tool-versions in all projects.

Probably worth scrapping the tool-versions file in this case and just using the latest versions, since they're unlikely to be backwards-incompatible.

Reply:
Ok, no worries, add whatever you need to the gitignore